packages/clarity-js/src/data/baseline.ts

```diff
@@ -14,33 +14,29 @@ export function start(): void {
 export function reset(): void {
     // Baseline state holds the previous values - if it is updated in the current payload, reset the state to current value after sending the previous state
     state = update ? { time: time(), event: Event.Baseline, data: buffer } : state;
-    buffer = buffer ? buffer : { visible: BooleanFlag.True, docWidth: 0, docHeight: 0, scrollX: 0, scrollY: 0, pointerX: 0, pointerY: 0 };
+    buffer = buffer ? buffer : { visible: BooleanFlag.True, docWidth: 0, docHeight: 0, screenWidth: 0, screenHeight: 0 };
     update = false;
 }
 
-export function visibility(visible: string): void {
-    buffer.visible = visible === "visible" ? BooleanFlag.True : BooleanFlag.False;
-    update = true;
-}
-
-export function track(event: Event, x: number, y: number): void {
+export function track(event: Event, width: number, height: number): void {
     switch (event) {
         case Event.Document:
-            buffer.docWidth = x;
-            buffer.docHeight = y;
-            break;
-        case Event.Scroll:
-            buffer.scrollX = x;
-            buffer.scrollY = y;
+            buffer.docWidth = width;
+            buffer.docHeight = height;
             break;
-        default:
-            buffer.pointerX = x;
-            buffer.pointerY = y;
+        case Event.Resize:
+            buffer.screenWidth = width;
+            buffer.screenHeight = height;
             break;
     }
     update = true;
 }
 
+export function visibility(visible: string): void {
+    buffer.visible = visible === "visible" ? BooleanFlag.True : BooleanFlag.False;
```

Comment:
we would need time for last visibility state - otherwise we can't compute the visbile duration

Reply:
Clarified offline.